src/Symfony/Component/Intl/Data/Bundle/Writer/TextBundleWriter.php

```diff
@@ -189,10 +189,6 @@ private function writeArray($file, array $value, int $indentation)
      */
     private function writeTable($file, iterable $value, int $indentation, bool $fallback = true)
     {
-        if (!\is_array($value) && !$value instanceof \Traversable) {
-            throw new UnexpectedTypeException($value, 'array or \Traversable');
```

Comment:
like everything that is not removing obsolete phpdocs and adding type hints, this should be done in a lower branch

Reply:
@xabbuh has added the `terable` type-hint on 4.4 recently, but apparently he did not remove the obsolete `if` block. There might be other leftovers like this.

Reply:
Fixed in #33018 and I've searched for more of those checks. There is only this one, `\Symfony\Component\Console\Command\Command::setAliases` which has been fixed in master and a few in the Cache component which we can't remove because of the psr cache interface.